share/lib/config.js

```diff
@@ -515,6 +523,44 @@ const COLUMN_INDEX = {
       v: _ => _['OverHealPct'],
       f: _ => _ && _.replace? _.replace('%', '<small>%</small>') : '---'
     },
+    effective_per_second: {
+      v: _ => {
+        return _['enchps'] * (1 - parseInt(_['OverHealPct']) / 100)
+      },
+      f: (_, conf) => {
+        _ = pFloat(_)
+        return isNaN(_)?
+          '0'
+        : formatDps(_, conf.format, 'hps')
+      }
+    },
+    effective_pct: {
+      v: (_, players, encounter) => {
+        // Calculated and injected during Data.update()
+        return Math.round(_.effective_healing / encounter.rhealing_effective * 100)
```

Comment:
'calculateMax' exists for preventing render functions from require whole encounter data. if there's additional column which requires whole encounter access, please move into `Data.get()`.

Reply:
Just to clarify, you mean I should calculate that value in `Data.get()` and and store it directly in the player object ?

Reply:
yes, I would prefer that way, but it's ok to keep if current approach is better (sorry, i'm confused of my old current codebase)